pkg/utils/helm/utils.go

```diff
@@ -64,9 +64,10 @@ func InstallRelease(name string, namespace string, valueFile string, chartName s
 	return err
 }
 
-/**
-* check if the release exist
- */
+
+// CheckRelease checks if the release derived from given name and namespace exist.
+// It returns a bool variable exist and any error encountered.
+// If the release doesn't exist, the bool variable exist will be false.
```

Comment:
This two lines are unnecessary. I think Line 68 is self-explained enough

Reply:
I have deleted 69 and 70 lines.